app/views/categories/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="flex flex-col h-screen justify-between">
+  <div>
+    <div class="navigation py-6 px-3 blue-nav-white-text">
+      <ul class="flex justify-between items-center">
+        <li class="w-10"><%= link_to image_tag('backarrow.png', alt: 'Back'), categories_path, class: 'nav-link' %></li>
+        <h2><strong><%= @category.name %></strong></h2>
+        <li><%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: "nav-link" %></li>
+      </ul>
+    </div>
+    <p style="color: green"><%= notice %></p>
+
+    <%= render @category %>
+    <section class="px-5">
+      <div id="purchases">
+        <% @purchases.each do |purchase| %>
+          <a href="<%= category_purchase_path(@category, purchase) %>" class="m-1">
+          <%= render "purchases/purchase", purchase: purchase %>
+          </a>
+        <% end %>
+      </div>
+    </section>
+  </div>
+
+  <section class="flex flex-col space-y-2 p-5">
+      <a href="<%= new_category_purchase_path(@category) %>" class="">
+        <button data-testid="new-purchase-button" class="drop-shadow-md p-3 w-full green-btn text-white rounded-md hover:bg-green-600 focus:outline-none focus:bg-green-600">
+          New purchase
+        </button>
+      </a>
```

Comment:
- [ ] Please as per the requirement, ensure that this button is named **Add a new transaction** and that it is at the bottom of the page.  So, I would suggest that you style it the same way you styled the **Add a New Category** on the categories page.

### Requirement :point_down: 
![Screenshot from 2023-09-14 22-53-08](https://github.com/Salimer/Budget-app/assets/89228743/0b0b9e0a-edd5-476e-9450-ca53fcbaa41b)